docs/README-windows.md

```diff
@@ -0,0 +1,3 @@
+# Web3.py on Windows
+
+## Developer Setup (TODO)
```

Comment:
I think the "TODO" will have to move to its own line, or else it breaks the link target.